File: main/src/main/java/net/fwitz/math/main/binary/split/functions/examples/simple/SplitZ2.java
package net.fwitz.math.main.binary.split.functions.examples.simple;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitZ2 {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("z^2")
                .computeFn(SplitComplex::pow2)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/examples/simple/SplitZ2.kt
package net.fwitz.math.main.binary.split.functions.examples.simple

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitZ2 {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("z^2")
        .computeFn(SplitComplex::pow2)
        .render()
}